ashlar-server/ashlar_server/settings.py

```diff
@@ -0,0 +1,103 @@
+"""
+Django settings for ashlar_server project.
+
+Generated by 'django-admin startproject' using Django 1.8.6.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/1.8/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/1.8/ref/settings/
+"""
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+import os
+
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+# Retrieve secret variables from the environment
+SECRET_KEY = os.environ.get('SECRET_KEY', 'extra-secret')
```

Comment:
Good call pulling these from the environment; we often use `env_file` in concert with this pattern to provide an easy way for people to inject these variables. https://docs.docker.com/compose/compose-file/#env_file Usually we just put in a template file into the repo, e.g. `.env.example` that people can use by filling out the variables with their own values.

Reply:
Nice, I like that pattern. Even though it's not necessary yet, I'll add an .env file to the repo as part of this PR for completeness.